docs/modules/reference/pages/provisioning/detectors/JdbcDetector.adoc

```diff
@@ -0,0 +1,71 @@
+= JDBC Detector
+
+The JDBCDetector connects to a JDBC-enabled database and tries to fetch its catalog.
+If the fetch is successful, it returns the `detected` value.
+If it is not successful, it returns the `not detected` value.
+
+== Detector facts
+
+[options="autowidth"]
+|===
+| Implementation | `org.opennms.netmgt.provision.detector.jdbc.JdbcDetector`
+|===
+
+== Configuration and use
+
+The JDBCDetector uses the xref:reference:provisioning/detectors.adoc#ref-detector-provisioning-common-parameters[common configuration parameters].
+
+.Parameters for the JDBCDetector
+[options="header, autowidth"]
+[cols="1,4,1"]
+|===
+| Parameter
+| Description
+| Default value
+
+3+| *Required*
+
+| dbDriver
+| Database driver class name.
+| org.postgresql.Driver
+
+| password
+| JDBC password.
+| Empty String object.
+
+| url
+| JDBC URL to query.
+Note: this URL can differ based on JDBC drivers, databases, and so on.
+| `jdbc:postgresql://localhost/opennms`
+
+| user
+| JDBC username.
+| postgres
+
+3+| *Optional*
+
+| port
+| Port to query.
+| 3306
+|===
+
+== Database driver
+The JDBCDetector requires a JDBC driver to communicate with any database.
+Since {page-component-title} uses a PostgreSQL database, the PostgreSQL JDBC driver is included out of the box.
+For all other database systems, you must provide a compatible JDBC driver as a JAR file.
+To provide a JDBC driver, place the driver-jar in your `$\{OPENNMS_HOME}/lib` folder.
+
+== Examples
+
+[source,xml]
+----
+<detector name="MySQL-JDBC" class="org.opennms.netmgt.provision.detector.jdbc.JdbcDetector">
+    <parameter key="password" value="borkborkbork"/>
+    <parameter key="user" value="opennms"/>
+    <parameter key="dbDriver" value="com.mysql.jdbc.Driver"/>
+    <parameter key="url" value="jdbc:mysql://OPENNMS_JDBC_HOSTNAME:3306/test"/>
+    <parameter key="timeout" value="5000"/>
+</detector>
+----
+
+This example assumes you have loaded the MySQL driver file.
```

Comment:
Could / Should this link to the documentation for doing so? e.g. deployment/minion/installing-jdbc-driver.adoc (similar documentation should also exist for core, someday?)

Reply:
Let's do it! And I'll create a Jira to add similar docs for the Core.

Reply:
```suggestion
This example assumes you have loaded the MySQL driver file (see xref:deployment:minion/installing-jdbc-driver.adoc[Install JDBC Driver]).
```